### ISSUE TITLE
Sector stuck in PackingFailed + ERROR: normal shutdown of state machine

### ISSUE THREAD

**Issue by RobQuistNL** [filecoin-project/lotus#3354]
```
$ lotus-miner sectors status --log 12
SectorID:	12
Status:		PackingFailed
CIDcommD:	<nil>
CIDcommR:	<nil>
Ticket:		31d2af880257367ecc3df142ea843f175b14b94efb28b1a5175eca4e83c7095b
TicketH:	3541
Seed:		
SeedH:		0
Proof:		
Deals:		[6651 7187 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0]
Retries:	0
--------
Event Log:
0.	2020-08-25 21:55:07 +0000 UTC:	[event;sealing.SectorStart]	{"User":{"ID":12,"SectorType":3}}
1.	2020-08-25 22:55:07 +0000 UTC:	[event;sealing.SectorStartPacking]	{"User":{}}
2.	2020-08-26 00:44:00 +0000 UTC:	[event;sealing.SectorAddPiece]	{"User":{"NewPiece":{"Piece":{"Size":131072,"PieceCID":{"/":"baga6ea4seaqhcplgn52r2465czdlkghnl427j6wd5mrirmxmd5wlp4yt7i47iiq"}},"DealInfo":{"DealID":6651,"DealSchedule":{"StartEpoch":5468,"EndEpoch":706489},"KeepUnsealed":true}}}}
3.	2020-08-26 00:44:00 +0000 UTC:	[event;sealing.SectorAddPiece]	{"User":{"NewPiece":{"Piece":{"Size":131072,"PieceCID":{"/":"baga6ea4seaqg7ov3yuishamnlpwnmg2e65n5h3iu6aclzaezt3udntadszabufa"}},"DealInfo":{"DealID":7187,"DealSchedule":{"StartEpoch":5679,"EndEpoch":709369},"KeepUnsealed":true}}}}
4.	2020-08-26 10:31:03 +0000 UTC:	[event;sealing.SectorStartPacking]	{"User":{}}
5.	2020-08-26 11:00:30 +0000 UTC:	[event;sealing.SectorPacked]	{"User":{"FillerPieces":[{"Size":262144,"PieceCID":{"/":"baga6ea4seaqhklmwsp5bm5jehfkhnyyxvgcyb4aji6x3piyfidlclkjjdtasuby"}},{"Size":524288,"PieceCID":{"/":"baga6ea4seaqhaixwb57pnlp2c4ixuutbtyym5kbmnadvvxy4mz3yn3cqn3xs2gi"}},{"Size":1048576,"PieceCID":{"/":"baga6ea4seaqntgehxfzvoouw4ejzgzcsg3axwh2moa2noi6hvgpxbg5u3jqrmky"}},{"Size":2097152,"PieceCID":{"/":"baga6ea4seaqnbnjq3oylj4s4luxsukg752aiwu2bfibjghyyysm7lisubbvrgjq"}},{"Size":4194304,"PieceCID":{"/":"baga6ea4seaqijqccdoqgqwqbx54vui2eazh6ijf5kku5eq3xwokp6tclivuoqei"}},{"Size":8388608,"PieceCID":{"/":"baga6ea4seaqgl4u6lwmnerwdrm4iz7ag3mpwwaqtapc2fciabpooqmvjypweeha"}},{"Size":16777216,"PieceCID":{"/":"baga6ea4seaqkejdvbaufquewln7dgszre6ymaqvr2bdnyvcaee3we7gypgoocoq"}},{"Size":33554432,"PieceCID":{"/":"baga6ea4seaqnv7nlnwutmrctyjwtg4tlt7x6gq56r6awjhwabgvnh6x7kbqxkca"}},{"Size":67108864,"PieceCID":{"/":"baga6ea4seaqnsqov4dldcsuzlqz77pkpxzurddlt2ts72lgtd4hxzbxl3ukoobq"}},{"Size":134217728,"PieceCID":{"/":"baga6ea4seaqfctcdlq6qju2juu3f7pkz77drgyurcf4fteobupctv4rapf2buly"}},{"Size":268435456,"PieceCID":{"/":"baga6ea4seaqk2bufhfu5g7ju74eobh2wsmfevum2rhppmdf75z7b2m4byhtryny"}},{"Size":536870912,"PieceCID":{"/":"baga6ea4seaqdsvqopmj2soyhujb72jza76t4wpq5fzifvm3ctz47iyytkewnubq"}},{"Size":1073741824,"PieceCID":{"/":"baga6ea4seaqmzq6acl23axubdiv37xipnaz3qqtvwr57ekoaauviescphqnfwpi"}},{"Size":2147483648,"PieceCID":{"/":"baga6ea4seaqh34u3nf3tdgpi6k2aw54rtucikcpo25uofrzjpmprinydj7b4mla"}},{"Size":4294967296,"PieceCID":{"/":"baga6ea4seaqgntqfunthkuwpixacxtcoqojjdg66vq254l7vmjyyjdu7pntvcby"}},{"Size":8589934592,"PieceCID":{"/":"baga6ea4seaqnqyicdbbfvnpjlmokmi45fgroiigxa2uw6nz6f6ojveoxlhizwai"}},{"Size":17179869184,"PieceCID":{"/":"baga6ea4seaqg2nsld34emra2ljfgrbrdcswmbjdpaftrpzjuipudt3w7qpbikpa"}}]}}
6.	2020-08-26 20:12:09 +0000 UTC:	[event;sealing.SectorPreCommit1]	{"User":{"PreCommit1Out":"eyJyl....F19","TicketValue":"MdK...PHCVs=","TicketEpoch":3541}}
7.	2020-08-26 23:24:26 +0000 UTC:	[event;sealing.SectorRestart]	{"User":{}}
8.	2020-08-26 23:48:28 +0000 UTC:	[event;sealing.SectorRestart]	{"User":{}}
9.	2020-08-26 23:50:55 +0000 UTC:	[event;sealing.SectorSealPreCommit2Failed]	{"User":{}}
	seal pre commit(2) failed: presealing sector 12 (): Rust panic: no unwind information
10.	2020-08-26 23:51:55 +0000 UTC:	[event;sealing.SectorRetrySealPreCommit2]	{"User":{}}
11.	2020-08-27 00:17:25 +0000 UTC:	[event;sealing.SectorSealPreCommit2Failed]	{"User":{}}
	seal pre commit(2) failed: presealing sector 12 (): Rust panic: no unwind information
12.	2020-08-27 00:18:25 +0000 UTC:	[event;sealing.SectorRetrySealPreCommit1]	{"User":{}}
13.	2020-08-27 00:18:25 +0000 UTC:	[event;sealing.SectorPackingFailed]	{"User":{}}
3:15
```
Its stuck in `PackingFailed`, can't change the state to `PreCommit1` or `PreCommit2` because:

```
$ lotus-miner sectors update-state --really-do-it 12 Packing
ERROR: normal shutdown of state machine
```

**Comment by benjaminh83** [comment]
Yep, It's not really possible to do anything: 
`90: Packing     sSet: NO  active: NO  tktH: 0    seedH: 0    deals: [16949]                                                                        toUpgrade:false`

And: 
`lotus-miner sectors update-state --really-do-it 90 Faulty

ERROR: normal shutdown of state machine
`

**Comment by RobQuistNL** [comment]
Also tried removing:
```
lotus-miner sectors remove --really-do-it 12
ERROR: normal shutdown of state machine
```

**Comment by Reiers** [comment]
**Hi @RobQuistNL** 

Thanks for the report. This is fixed in the latest release.
Please upgrade Lotus.

Link to latest tags [here.](https://github.com/filecoin-project/lotus/tags)

```
v1.14.4 -  SPs should be able to force
update sectors status without getting blocked by normal shutdown of state machine.
```

Closing Lotus issue ticket.

If you are still experiencing problems, open a new ticket or ask for help in our [Discussions forum.](https://github.com/filecoin-project/lotus/discussions)
